src/org/parosproxy/paros/network/HttpMessage.java

```diff
@@ -479,15 +480,27 @@ public boolean equalType(HttpMessage msg) {
 	    return result;
 	}
 	
+	/**
+	 * Compares the parameter names used in GET and POST messages. For POST

```

Comment:
Could you add this explanation to `equalType` too? That's the method the users of the class will use/see (without checking the source).